internal/services/oracle/validate/cloud_vm_cluster.go

```diff
@@ -115,3 +115,33 @@ func SystemVersion(i interface{}, k string) (warnings []string, errors []error)
 
 	return
 }
+
+func DatabaseServerType(i interface{}, k string) (warnings []string, errors []error) {
+	v, ok := i.(string)
+	if !ok {
+		errors = append(errors, fmt.Errorf("expected type of %s to be string", k))
+		return
+	}
+
+	if len(v) < 1 || len(v) > 255 {
+		errors = append(errors, fmt.Errorf("%s must be %d to %d characters", k, 1, 255))
+		return
+	}
+
+	return
+}
+
+func StorageServerType(i interface{}, k string) (warnings []string, errors []error) {
```

Comment:
We have a validation helper function that does the job of these functions if the only validation we can do is string length?

https://github.com/hashicorp/terraform-provider-azurerm/blob/39033d7f1ae63a2a283bc2c65a2f84058366ccce/internal/tf/validation/pluginsdk.go#L278